ga4gh/serverconfig.py

```diff
@@ -45,6 +58,38 @@ class ProductionConfig(BaseConfig):
     DATA_SOURCE = None
 
 
+class GoogleOIDCConfig(ProductionConfig):
```

Comment:
I tried a superclass and some multiple inheritance for specific instances, but it seemed to be just creating confusion. Have gone back to single inheritance, but have cleaned things up a bit.